operator/cmd/mesh/testdata/manifest-generate/output/ingressgateway_k8s_settings.yaml

```diff
@@ -27,6 +27,8 @@ spec:
         heritage: Tiller
         release: istio
         chart: gateways
+        service.istio.io/canonical-name: istio-ingressgateway
+        service.istio.io/canonical-revision: "1.5"
```

Comment:
what is the intent of this label? Do you want this to correspond to istio version? I was expecting this to say "latest"

Reply:
According to the [Istio Canonical Service proposal](https://docs.google.com/document/d/1EsKpYNW55-LqG08zdjp1tAd224T9IEG3CEnusmL-Joc/edit):

> Provide a mechanism to subdivide groups into revisions if revision information is present.

Using "latest" is to use the fallback value for these services. I guess that is fine, but I figured that Istio should use a definitive version label when possible. For `release-1.5`, using 1.5 seemed appropriate. But I have no strong feelings here.